recipes/ot-commissioner/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+import os
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+
+
+class OtCommissionerConan(ConanFile):
+    name = "ot-commissioner"
+    description = "OpenThread Commissioner, a Thread commissioner for joining new Thread devices and managing Thread networks."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://giþthub.com/openthread/ot-commissioner"
+    topics = ("conan", "thread", "commissioning")
+    license = "MIT"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False]}
+    default_options = {"shared": False,
+                       "fPIC": True}
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("libevent/2.1.11")
+        self.requires("mdns/20200331")
+        self.requires("nlohmann_json/3.7.3")
+        self.requires("mbedtls/2.16.3-gpl")
+        self.requires("fmt/6.1.2")
+        # TODO: port to CCI
+        self.requires("cose-c/20200225@gocarlos/testing")
```

Comment:
```suggestion
        self.requires("cose-c/cci.20200430")
```

Hopefully, cose-c is already in CCI, so let's try to use it.